sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -141,6 +141,7 @@ def tune(
         base_image: str = constants.BASE_IMAGE_TENSORFLOW,
         namespace: Optional[str] = None,
         algorithm_name: str = "random",
+        algorithm_settings: Union[dict, list[models.V1beta1AlgorithmSetting], None] = None,
```

Comment:
Should we use `List` similar as here: https://github.com/kubeflow/katib/blob/2798d300e2323b0cddcc64f2e55a3142df126585/sdk/python/v1beta1/kubeflow/katib/api/katib_client.py#L146C34-L146C38 ?

Reply:
Fixed!